src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -1850,6 +1844,31 @@ export default class AircraftModel {
         return headingToWaypoint;
     }
 
+    /**
+     * Determine if it is time to move to the next fix.
+     *
+     * @for AircraftModel
+     * @method _isReadyToMoveToWaypoint
+     * @param waypoint {WaypointModel}
+     * @return {boolean}
+     */
+    _isReadyToMoveToWaypoint(waypoint) {
```

Comment:
Is this checking whether or not it is ready to move _past_ the provided waypoint?

Reply:
It checks whether the aircraft is close enough to the waypoint to consider it reached. So maybe `hasReachedWaypoint` would be a better name.

Reply:
Better, though I would say that `.hasReachedWaypoint()` has the implication that it would return true for all waypoints which have been passed (which it won't do). Maybe like `.isPassingWaypoint()` or something? That's not great, but is in the right direction I think.

Reply:
done